src/legacy/core_plugins/data/public/filter/filter_bar/filter_editor/lib/filter_label.tsx

```diff
@@ -23,7 +23,12 @@ import { i18n } from '@kbn/i18n';
 import { existsOperator, isOneOfOperator } from './filter_operators';
 import { esFilters } from '../../../../../../../../plugins/data/public';
 
-export function getFilterDisplayText(filter: esFilters.Filter, filterDisplayName: string) {
+interface Props {
+  filter: esFilters.Filter;
+  valueLabel: string;
```

Comment:
In the filter label component [test](https://github.com/elastic/kibana/pull/50743/files#diff-b7869d14e1b93a0ccbfb0ff4d9ea29e6R33), the `valueLabel` is not provided but it isn't optional in the Props interface here. Perhaps we should be adding `undefined` to the valueLabel type?